option optimize_for = LITE_RUNTIME;

message PetDressRequest
{
 required uint32 pos = 1;
 required uint32 posZhengXing = 2;
}
